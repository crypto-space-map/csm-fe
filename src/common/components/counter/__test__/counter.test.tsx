import { render } from '@testing-library/react';

import { Counter } from '..';

describe('<Counter />', () => {
  it('should match snapshot', () => {
    const { container } = render(<Counter>11</Counter>);
    expect(container).toMatchSnapshot();
  });
  it('should be defined', () => {
    const { container } = render(<Counter>12</Counter>);
    expect(container).toBeDefined();
  });
});
