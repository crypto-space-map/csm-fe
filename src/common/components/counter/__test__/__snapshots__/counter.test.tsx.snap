// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Counter /> should match snapshot 1`] = `
<div>
  <span
    class="css-1vx789d"
  >
    11
  </span>
</div>
`;
